src/appengine/handlers/testcase_detail/show.py

```diff
@@ -86,7 +86,7 @@ def highlight_common_stack_frames(crash_stacktrace):
   stack_trace_line_format = '^ *#([0-9]+) *0x[0-9a-f]+ (.*)'
```

Comment:
This field is not added in data_types.py as a generic configuration attribute, not also updated in go datastore model.

Reply:
Marking resolved since we plan to use the default environment to update these.